src/controllers/BoardContainer.js

```diff
@@ -78,10 +79,18 @@ class BoardContainer extends Component {
             })
           case 'UPDATE_CARDS':
             return actions.updateCards({laneId: event.laneId, cards: event.cards})
+          
           case 'UPDATE_LANES':
             return actions.updateLanes(event.lanes)
           case 'UPDATE_LANE':
             return actions.updateLane(event.lane)
+          case 'MOVE_LANE':
+            return actions.moveLane({ oldIndex: event.fromIndex, newIndex: event.toIndex })
```

Comment:
I know `fromIndex` and `toIndex`names are better then `oldIndex` and `newIndex`. But it is preferable not to create new entities. We need either rename current attributes either use their. As we can't simply rename their because of using them by users we just have to use current names.

@LucasBadico What do you think?

Reply:
make sense. Principle of least surprise.